src/components/AdminCampaignList/CampaignTable.jsx

```diff
@@ -278,7 +294,6 @@ export class CampaignTable extends React.Component {
     const { campaigns, pageInfo } = this.props.data.organization.campaigns;
     const { limit, offset, total } = pageInfo;
     const displayPage = Math.floor(offset / limit) + 1;
-    console.log("CampaignTable", campaigns);
```

Comment:
Should this stay in the code? I thought it was from my own debugging but apparently it was here before!

Reply:
I'm a big fan of removing console logs, but I also see them all over the codebase so I'm not sure. @schuyler1d ?

Reply:
I don't want a concerted efforts to delete console.logs (yet -- we have some plans around swapping out logging frameworks for better flexibility), we can delete specific instances, especially if they're annoying.

Anyway, this delete is good and fine.